openedx/core/djangoapps/oauth_dispatch/dot_overrides/validators.py

```diff
@@ -91,6 +92,23 @@ def save_bearer_token(self, token, request, *args, **kwargs):
         request.grant_type = grant_type
         request.user = user
 
+    def get_default_scopes(self, client_id, request, *args, **kwargs):
+        """
+        Returns the default scopes.
+
+        If the request payload does not have `scopes` attribute for a grant_type of
+        client credentials, add `user_id` as a default scope if it is an allowed scope.
+        """
+        default_scopes = super().get_default_scopes(client_id, request, *args, **kwargs)
+        if settings.FEATURES.get('ENABLE_USER_ID_SCOPE', True):
```

Comment:
1. This doesn't need to live in FEATURES, and could be top-level, but not a huge deal for a temporary toggle.
2. I would rename to something more specific like ENABLE_DEFAULT_ALLOWABLE_USER_ID_SCOPE.
3. You could use a `SettingToggle`. Again, your choice on updating, but that is the best practice.
4. The toggle should be documented via annotations. See other SettingToggles in edx-platform and https://edx.readthedocs.io/projects/edx-toggles/en/latest/how_to/documenting_new_feature_toggles.html.